docs/whatsnew/7.0.rst

```diff
@@ -1,4 +1,4 @@
 :orphan:
 
 `What's New in Astropy 7.0?
-<https://docs.astropy.org/en/v7.0.0/whatsnew/7.0.html>`__
+<https://docs.astropy.org/en/v7.0.x/whatsnew/7.0.html>`__
```

Comment:
@astropy/astropy-project-release-team , do we need a follow up issue to change this back to tagged version after release?